### ISSUE TITLE
Different calculation for the weight of concepts

### ISSUE THREAD

**Issue by nbonfils** [nbonfils/bibliograph2#15]
[OpenAlex gives us a score](https://docs.openalex.org/api-entities/works/work-object#concepts) for each concepts linked to a publication.
The score representing how relevant a concept is to a publication, instead of counting every concepts as equal with regards to the same publication, we could compute a slightly more involved weight for the concepts rather that the number of times they appear in the corpus.

What we get in the concepts:
![openalexconcepts](https://github.com/nbonfils/bibliograph2/assets/14347927/aecce163-c382-4255-ad69-3fe5c8180e10)

**Comment by tommv** [comment]
Yes very good idea, let's try to filter out all the concepts below a 0.5 threshold
